### PR TITLE
AspNet3P8

### PR DESCRIPTION
**Pull request by VaclavK**

### CODE REVIEW

**Comment by VaclavK** [comment]
@arturek @dguerin let's review this structurally before/in parallel to  fixing the build

File: src/Eshopworld.Web/Eshopworld.Web.csproj [hunk 1]
```diff
@@ -22,22 +22,27 @@
...
-    <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="2.2.4" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="2.2.0" />
-    <PackageReference Include="Microsoft.ServiceFabric.Actors" Version="3.4.641" />
-    <PackageReference Include="Microsoft.ServiceFabric.AspNetCore.Kestrel" Version="3.4.641" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
-    <PackageReference Include="System.Reactive" Version="4.2.0-preview.102" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="3.0.0-preview8.19405.4" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.0.0-preview8.19405.4" />
```
Comment (arturek) [inline src/Eshopworld.Web/Eshopworld.Web.csproj]: this and some other packages should be a part of Microsoft.AspNetCore.App

Comment (dguerin) [reply src/Eshopworld.Web/Eshopworld.Web.csproj]: question I have is do we need all the packages in Microsoft.AspNetCore.App - in previous versions this brought in a lot of packages that were not needed - which ended up bloating out the package

**Comment by VaclavK** [comment]
preview1 released to myget- not currently merging to master on account of being preview

File: src/Eshopworld.Web/Eshopworld.Web.csproj (hunk 1 above)
Comment (dguerin) [reply src/Eshopworld.Web/Eshopworld.Web.csproj]: thanks for the info @VaclavK

**Pull request closed without merging**